task/acs-deploy-check/0.1/acs-deploy-check.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: acs-deploy-check
+spec:
+  description: >-
+    Policy check a deployment with StackRox/RHACS This tasks allows you to check
+    a deployment against build-time policies and apply enforcement to fail
+    builds. It's a companion to the stackrox-image-scan task, which returns full
+    vulnerability scan results for an image.
+  params:
+    - name: rox-central-endpoint
+      type: string
+      description: |
+        The address:port tuple for StackRox Central.
+        For example: rox.stackrox.io:443
+    - name: rox-api-token-secret-name
+      type: string
+      description: |
+        Secret containing the StackRox API token with CI permissions under rox-api-token key
+    - name: verbose
+      type: string
+      default: 'true'
+    - name: insecure-skip-tls-verify
+      type: string
+      default: 'false'
+      description: |
+        When set to `"true"`, skip verifying the TLS certs of the Central
+        endpoint.  Defaults to `"false"`.
+  results:
+    - description: Output of `roxctl deployment check`
+      name: check_output
+  volumes:
+    - name: repository
+      emptyDir: {}
+    - name: rox-secret
+      secret:
+        secretName: $(params.rox-api-token-secret-name)
+        optional: true
+  steps:
+    - name: rox-deploy-scan
+      image: 'registry.access.redhat.com/ubi8:8.7-1026'
+      volumeMounts:
+        - name: repository
+          mountPath: /workspace/repository
+        - name: rox-secret
+          mountPath: /rox-secret
+      workingDir: /workspace/repository
+      script: |
+        #!/usr/bin/env bash
+
+        # Check if rox API token is configured
+        if test -f /rox-secret/rox-api-token ; then
+          export ROX_API_TOKEN=$(</rox-secret/rox-api-token)
+        else
+          echo "rox API token is not set, demo will exit with success"
+          echo "TODO: configure the pipeline to have access to ROXCTL"
+          exit 0
+        fi
+
+        echo "Using rox central endpoint $(params.rox-central-endpoint)"
+
+        echo "Configuration"
+        echo "Directory = $(pwd)"
+        ls -al
+
+        curl -s -k -L -H "Authorization: Bearer $ROX_API_TOKEN" \
```

Comment:
Is this the best way to get the CLI? Maybe use an image that already has the roxctl? There is no guarantee the version being fetched is the version expected.